packages/kbn-expect/expect.js

```diff
@@ -98,6 +98,9 @@ Assertion.prototype.assert = function (truth, msg, error, expected) {
 
   if (!ok) {
     err = new Error(msg.call(this));
+    if (this.customMsg) {
+      err.message = '< ' + this.customMsg + ' > ' + err.message;
```

Comment:
I think we should just replace the message instead of combining them.

Reply:
As in:

```suggestion
- Error: < Old filter should be removed > expected true to equal false
+ Error: Old filter should be removed
```

?

Reply:
Yeah, that's what I'm thinking

Reply:
Fixed. I agree it is better to replace message. @spalger Any other suggestions?

Reply:
Actually, I don't fully agree. For a boolean decision this makes sense, as the `customMsg` probably says enough. But for other validations it might still be valuable to see the `actual` value in the error message.

Reply:
Example:
![image](https://user-images.githubusercontent.com/1945390/67409985-abb58400-f5bb-11e9-9351-d25b182f5e91.png)

If it would just tell me `Error: Count details should be as expected`, I would lose a lot of value from the message compared to what it currently shows.

Reply:
@pheyos I think the point is that you can specify expected/actual in your custom message the way you see it.

Reply:
@dmlemeshko yes, but for object comparison I wouldn't easily get that nice highlight what detail doesn't match. But if you all think this is the wanted level of details, I can live with it as well.

Reply:
Well, there's still the option to not add a custom message and keep the old output if needed, so LGTM

Reply:
Yeah, I don't think there's a lot of value in overriding the message in an object comparison, unless you're trying to hide the actual/expected message (maybe it's too large or something).